quickproject.lisp

```diff
@@ -2,22 +2,20 @@
 
 (in-package #:quickproject)
 
-(defvar *name*)
-(setf (documentation '*name* 'variable)
-      "The name of the project currently being created.")
+(defvar *name* nil
```

Comment:
This is semantically different.

Reply:
Oh I see, it does not default to `nil` but `#<unbound-variable *name*>`.